examples/mixed-spmv/doc/kind

```diff
@@ -0,0 +1 @@
+basic
```

Comment:
This could go under `techniques` (advanced techniques) or we could create a new category for `mixed precision` in `doc/scripts/examples.pl:193`.